### PR TITLE
server: move client function to util

### LINKED ISSUE #5837: Make the service start with independent mode
## Development Task

- [ ] parse mode #5858
- [x] #5857
- [x] #5921
- [ ] #5933
- [ ] #5836
- [ ] #5956
- [ ] run independent mode with resource manager

### PR DESCRIPTION
**Pull request by lhy1024**
Signed-off-by: lhy1024 <admin@liudos.us>

### What problem does this PR solve?

Issue Number: Close #5837

### What is changed and how does it work?

```commit-message
```

### Check List

Tests

- Unit test

### Release note

```release-note
 None.
```

### CODE REVIEW

**Comment by lhy1024** [comment]
cc @rleungx @binshi-bing

File: pkg/utils/etcdutil/etcdutil.go [hunk 1]
```diff
@@ -36,6 +36,9 @@ import (
 )
 
 const (
+	// DefaultEtcdClientTimeout is the default timeout for etcd client.
+	DefaultEtcdClientTimeout = 3 * time.Second
```
Comment (rleungx) [inline pkg/utils/etcdutil/etcdutil.go:40]: Do we need to expose it?

**Review by rleungx — APPROVED** [review]
The rest LGTM, please also change the PR title.

File: pkg/utils/etcdutil/etcdutil.go (hunk 1 above)
Comment (lhy1024) [reply pkg/utils/etcdutil/etcdutil.go:40]: I am not sure whether is used by other servers.

Comment (rleungx) [reply pkg/utils/etcdutil/etcdutil.go:40]: Once it will be used, we can expose it then.

Comment (lhy1024) [reply pkg/utils/etcdutil/etcdutil.go]: done

**Review by HunDunDM — APPROVED** [review]

**Pull request merged**